backstop/scripts/ready.js

```diff
@@ -113,6 +113,33 @@ module.exports = async function (page, scenario, vp) {
     await page.waitForSelector('.cbFormErrorMarker', {visible: true})
   }
 
+  let leafletMapInitialized = await page.evaluate(async function () {
+    let initialized = undefined;
+    var containers = document.querySelectorAll(".js-leaflet-map");
+    if (containers.length) {
+      containers.forEach(function (e) {
+        var container = L.DomUtil.get(e);
+        if (container != null) {
+          if (container._leaflet_id == null) {
+            initialized = false;
+          }
+        }
+      });
+    }
+    initialized = true;
+  })
+
+  if (leafletMapInitialized == false) {
+    await page.waitForSelector('.js-leaflet-map .leaflet-tile-container', {
+      visible: true,
+      timeout: 10000,
```

Comment:
```suggestion
      timeout: 10000,
```
we are adding a 10-second timeout here? is that too much?

Reply:
That's the maximum time that we wait for an element to become visible, it is usually less then 2 second.